src/server/cli/load-config-file.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017-2022 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { loadFileSync } from "../utils/file/file";
+
+export function loadConfigFile(configPath: string): object {
+  try {
+    return loadFileSync(configPath, "yaml");
+  } catch (e) {
+    throw new Error("SOMETHING"); // TODO: something that commander can handle
```

Comment:
TODO: pass commander program reference and exit with proper error instead of throwing.